app/base/views.js

```diff
@@ -56,5 +57,8 @@ exports.login = function (req, res) {
 exports.logout = function (req, res) {
   req.logout();
   api.unset_token();
-  res.redirect('/');
+  req.session.destroy(err => {
```

Comment:
If `err` is not used in the handler, you may want to rename this to `_` to make it clear it's an unused argument.

Reply:
I have done that sometimes, but it's a Python (maybe Ruby) idiom and there's a (unfortunately named) popular Javascript library called [Underscore](http://underscorejs.org/) which uses the `_` character and muddies the waters.

Reply:
Also a Go idiom so almost a standard de-facto :D - I only said it as we have an `_` already in one of the files, but I imagine if underscore.js is also used it could cause a bit of confusion.